astrklp.c

```diff
@@ -0,0 +1,36 @@
+/*Sean Kee*/
+/*Half Diamond*/
+#include <stdio.h>
+
+int main()
```

Comment:
You could have implemented this more efficiently (in an O(n^2) time complexity). Good job completing the assignment though!